app/components/works/contributor_role_component.rb

```diff
@@ -52,8 +54,8 @@ def key
 
     # list for work_form pulldown
 
-    def grouped_options
-      %w[person organization].map do |key|
+    def grouped_options(contributor_type)
+      [contributor_type].map do |key|
         ContributorType.new(key:, citable: false)
       end
```

Comment:
Allows us to render just type of drop down at a time.  Yes, it is still a grouped select, with just a single group.  I like the way this displays though (it reinforces the "Individual" vs "Organization" header).